src/Joblify.Core.Data/Models/Offer.cs

```diff
@@ -0,0 +1,39 @@
+﻿using System.ComponentModel.DataAnnotations;
+using System.ComponentModel.DataAnnotations.Schema;
+
+namespace Joblify.Core.Data.Models
+{
+    public class Offer
+    {
+        [Key]
+        public int Id { get; set; }
+
+        [Required]
+        public string Title { get; set; }
+
+        public string Description { get; set; }
+
+        public float Price { get; set; }
+        
+        [Required]
+        [ForeignKey("Category")]
+        public int CategoryId { get; set; }
+
+        public Category Category { get; set; }
+
+        [ForeignKey("Trade")]
+        public int TradeId { get; set; }
+
+        public Trade Trade { get; set; }
+
+        public string AvailableTime { get; set; }
+
+        public bool IsDeleted { get; set; }
```

Comment:
Can we set state IsDeleted like false? Or is it a bad idea? @channel

Reply:
@Bartosz97  booleans are not nullable - by default they are false. (If I have understood you correctly 😄 )

Reply:
I'm a pitcher. These are the effects of working for 12 hours ;C.

Reply:
It's all right, go get some rest C: